boxes.yaml.example

```diff
@@ -44,6 +44,7 @@ centos7-bats-ci:
       bats_tests:
         - "swapfile.bats"
         - "fb-install-katello.bats"
```

Comment:
You mean basically completely dropping `fb-install-katello.bats`? Yeah we can do that.